A few errors to check/fix though: 
```
alma/entities/base.py:1:1: F401 'copy.deepcopy' imported but unused
alma/entities/base.py:9:12: E713 test for membership should be 'not in'
```

--- a/alma/entities/base.py
+++ b/alma/entities/base.py
@@ -1,12 +1,9 @@
-from copy import deepcopy
-
-
 class Base:
     def __init__(self, data):
         self._data = data
 
     def __getattr__(self, key):
-        if not key in self._data:
+        if key not in self._data:
             raise AttributeError(key)
         return self._data[key]
 

